docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1489,6 +1489,31 @@ How to **fix** it:
 - Ensure each compactor replica has successfully updated bucket index of each owned tenant within the double of `-compactor.cleanup-interval` (query below assumes the cleanup interval is set to 15 minutes):
   `time() - cortex_compactor_block_cleanup_last_successful_run_timestamp_seconds > 2 * (15 * 60)`
 
+### err-mimir-histogram-spans-buckets-mismatch
+
+This non-critical error occurs when Mimir receives a write request that contains a native (also known as sparse) histogram whose either negative or positive spans specify
+a different number of buckets than there are provided.
+
+> **Note**: Series with invalid histograms are skipped during the ingestion, and series within the same request are ingested.
+
+### err-mimir-histogram-span-negative-offset
+
+This non-critical error occurs when Mimir receives a write request that contains a native (also known as sparse) histogram which has a span whose offset is negative, making it overlap with the previous span.
+
+> **Note**: Series with invalid histograms are skipped during the ingestion, and series within the same request are ingested.
+
+### err-mimir-histogram-negative-bucket-count
+
+This non-critical error occurs when Mimir receives a write request that contains a native (also known as sparse) histogram which has a bucket whose observation count is negative.
+
+> **Note**: Series with invalid histograms are skipped during the ingestion, and series within the same request are ingested.
+
+### err-mimir-histogram-count-not-big-enough
+
+This non-critical error occurs when Mimir receives a write request that contains a native (also known as sparse) histogram whose overall observation count is lower than the number of observations found in the buckets.
```

Comment:
```suggestion
This is a non-critical error. It occurs when Mimir receives a write request that contains a native (also known as sparse) histogram whose overall observation count is lower than the number of observations found in the buckets.
```
This is a very long sentence. Is there a way to break down its complexity for non-native English speakers who have to store a lot of grammar in (human) memory?

Reply:
What is an "overall" observation count? Can we remove this term?